pkg/orchestrator/redact/pod_test.go

```diff
@@ -10,8 +10,77 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func TestScrubAnnotations(t *testing.T) {
```

Comment:
could you add a small test case where the annotation is missing?

Reply:
done in: https://github.com/DataDog/datadog-agent/pull/7933/commits/a5196f720e373cb024994d369b6c081c30545826